portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/CreateABTest.testcase

```diff
@@ -32,32 +30,31 @@ definition {
 
 		Navigator.openURL();
 
+		JSONGroup.addGroup(groupName = "Test Site Name");
+
 		// Connect and synchronyze with Analytics Cloud
 
-		AnalyticsCloud.syncAnalyticsCloud();
+		AnalyticsCloud.syncAnalyticsCloud(siteName = "Test Site Name");
 
 		// Create content page
 
-		SitePages.openPagesAdmin(siteURLKey = "guest");
-
-		ContentPages.addPage(pageName = "Content Page");
-
-		PageEditor.clickPublish();
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Content Page",
+			type = "content");
 
-		Navigator.openWithAppendToBaseURL(urlAppend = "web/guest/content-page");
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/content-page");
 	}
 
 	tearDown {
 		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
+		Refresh();
+
 		Click(locator1 = "ABTest#AB_TEST_BUTTON");
 
 		AnalyticsCloud.teardownAnalyticsCloud();
 
-		SitePages.openPagesAdmin(siteURLKey = "guest");
-
-		SitePages.deletePage(pageName = "Content Page");
-
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
```

Comment:
Things are not tearing down cleanly. I think this data pollution is what is causing **LocalFile.CreateABTest#CreateABTestByClickInvalidId** to fail. This screen shot is from the teardown section of **CreateABTest#CreateABTestByBounceRate**, which runs prior to **LocalFile.CreateABTest#CreateABTestByClickInvalidId**. 
![image](https://user-images.githubusercontent.com/2567802/114613201-dd128b80-9c57-11eb-97b5-76968dd02f26.png)

You'll need to ensure all the data is removed in the teardown. the `Click(locator1 = "ABTest#AB_TEST_BUTTON");` looks pretty vague, and you have no page navigation prior to this step, so it's possible it's on the wrong page?

This is the screenshot from the **LocalFile.CreateABTest#CreateABTestByClickInvalidId** failure on CI: 
![image](https://user-images.githubusercontent.com/2567802/114613697-8a859f00-9c58-11eb-9e15-71267da1108d.png)

It still has data from the previous "Bounce Rate" test, so is unable to create a new test on AC.